dice-engine/src/dice_engine/dice.clj

```diff
@@ -0,0 +1,40 @@
+(ns dice-engine.dice)
+
+(defn drop
+  "Returns a new map with :numeric-value updated after discarding the dice that match n."
+  [dice-roll & n]
+  ())
+
+(defn keep
+  "Returns a new map with :numeric-value updated after discarding the dice that do not match n."
```

Comment:
For the description, you should write the documentation a bit more high-level. Ex: "Takes a dice set, keeps <something> and returns a dice set". At least the first line should have a more straightforward explanation. Details like `:numeric-value` being updated is an implementation detail. Same comment for other similar functions

Reply:
I've updated the docstrings to have a more generic description